examples/kubernetes/templates/v1/cilium-containerd-ds.yaml.sed

```diff
@@ -131,6 +132,12 @@ spec:
         - mountPath: /tmp/cilium/config-map
           name: cilium-config-path
           readOnly: true
+        - mountPath: /lib/modules
+          name: lib-modules
+          readOnly: true
+        - mountPath: /sbin/modprobe
```

Comment:
Please add a comment to explain why it is needed

Reply:
Added.